apps/server/lib/lexical/server/provider/markdown.ex

```diff
@@ -0,0 +1,76 @@
+defmodule Lexical.Server.Provider.Markdown do
+  @moduledoc """
+  Utilities for formatting Markdown content.
+  """
+
+  alias Lexical.Protocol.Types.Markup
+
+  @type markdown :: String.t()
+
+  @doc """
+  Converts a string of Markdown into LSP markup content.
+  """
+  @spec to_content(markdown) :: Markup.Content.t()
+  def to_content(markdown) when is_binary(markdown) do
```

Comment:
I don't know if there is where to comment, but I've noticed that the markdown comment has a bunch of newlines at the end of it --enough to scroll the content off the limited area that emacs displays it.
You _do_ have to scroll to trigger this behavior, I thought I'd mention it.

Here's an example of it scrolled.

![Screenshot 2023-09-19 at 9 05 35 AM](https://github.com/lexical-lsp/lexical/assets/55809/5a238a96-fcc5-444e-883d-10cb1db20e45)

Reply:
This is super weird and I can't recreate it in a test. I'm thinking it might be a bug in the Emacs client? E.g. when I hover over `RemoteControl`, it's not scrollable:

![image](https://github.com/lexical-lsp/lexical/assets/503938/d63f6d9e-8d94-4667-8d12-24e0d5793a1d)

Reply:
a bug in `lsp_node`, no way. 🤣